build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```

Comment:
I read this article – [A Confusing Dependency](https://blog.autsoft.hu/a-confusing-dependency/)  – some time ago and since then, I always put `jcenter` last.

Reply:
👍 will change. thx